typescript/packages/common-identity/src/pass-key.ts

```diff
@@ -66,15 +71,15 @@ export class PassKey {
   // Different data is available within `PublicKeyCredentials` depending
   // on whether it was created or retrieved. We need the PRF assertion
   // only available on "get" requests, so we don't return a `PassKey` here.
-  static async create(name: string, displayName: string): Promise<void> {
+  static async create(name: string, displayName: string): Promise<PassKey> {
```

Comment:
This method originally returned a `PassKey`, but was changed back to `void` due to only receiving the PRF result in a GET command, leaving `PassKey` without any purpose during CREATE. Now that we track `id` for autoselecting a passkey during auth, we have a reason for create to return a `PassKey`.

With those changes, it'd be helpful if `createRootKey()`'s error message when `!prf` indicated that this PassKey was just created (if prf wasn't supported, it'd fail on creation), and a root key can only be created from a PassKey created via `PassKey.get`